src/MixedModels.jl

```diff
@@ -4,11 +4,11 @@ using Arrow
 using DataAPI
 using Distributions
 using GLM
+using LazyArtifacts
 using LinearAlgebra
 using Markdown
 using NLopt
 using Random
-using Pkg.Artifacts
```

Comment:
Can you drop the dependency on Pkg from Project.toml with this change?

Reply:
I think so, let's see if the bots agree. I also added a `compat` entry for LazyArtifacts